.drone.yml

```diff
@@ -68,14 +68,30 @@ pipeline:
     secrets:
       - google_key
     source: bundle
-    target: vic-ui-builds
+    target: vic-ui-builds/
     acl:
       - 'allUsers:READER'
     cache_control: 'public,max-age=3600'
     when:
       repo: vmware/vic-ui
       event: [push]
-      branch: [master, develop, 'releases/*']
+      branch: [master, develop]
+      status: success
+
+  publish-gcs-release-builds-on-pass:
+    image: 'victest/drone-gcs:1'
+    pull: true
+    secrets:
+    - google_key
+    source: bundle
+    target: vic-ui-builds/${DRONE_BRANCH}/
+    acl:
+    - 'allUsers:READER'
+    cache_control: 'public,max-age=3600'
+    when:
+      repo: vmware/vic-ui
+      event: [push]
+      branch: ['releases/*']
```

Comment:
It seems like builds from `develop` should probably be published to a subdirectory as well, so that only `master` builds are published in the top-level directory.